dotcom-rendering/src/components/marketing/epics/ContributionsEpic.tsx

```diff
@@ -114,6 +115,16 @@ const imageStyles = css`
 	object-fit: cover;
 `;
 
+const defaultTickerStylingSettings: TickerStylingSettings = {
```

Comment:
can this use the source model now?
https://github.com/guardian/csnx/blob/main/libs/%40guardian/source-development-kitchen/src/react-components/ticker/Ticker.tsx#L21